cloudflare/resource_cloudflare_argo_tunnel.go

```diff
@@ -55,10 +64,30 @@ func resourceCloudflareArgoTunnelCreate(d *schema.ResourceData, meta interface{}
 }
 
 func resourceCloudflareArgoTunnelRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*cloudflare.API)
+
+	attributes := strings.Split(d.Id(), "/")
+
+	if len(attributes) != 2 {
+		return fmt.Errorf("invalid id (\"%s\") specified, should be in format \"accountID/argoTunnelUUID\"", d.Id())
+	}
+
+	accID, tunnelID := attributes[0], attributes[1]
```

Comment:
this is `Import` specific behaviour, the `Read` doesn't need any of this. `Read` only syncs the remote API to the local state configuration.